packages/core/src/sdk/graphql/request.ts

```diff
@@ -23,3 +36,52 @@ export const request = async <Query = unknown, Variables = unknown>(
 
   return data
 }
+
+/* This piece of code was taken out of @faststore/graphql-utils */
+const baseRequest = async <V = any, D = any>(
+  endpoint: string,
+  { operation, variables, fetchOptions }: BaseRequestOptions<V>
+): Promise<GraphQLResponse<D>> => {
+  const { operationName, operationHash } = operation['__meta__']
+
+  if (!operationName) {
```

Comment:
This happens in the context of the running store, right? Should emit a warning (`console.warn`) with a better error message to the developer?

Reply:
This was a leftover from when I was debugging! Nice catch. I'll remove it.